rcl/test/rcl/test_events.cpp

```diff
@@ -269,6 +269,71 @@ wait_for_msgs_and_events(
   return RCL_RET_OK;
 }
 
+// @brief Conditional function for determining when the wait_for_msgs_and_events loop is complete.
+/**
+ * Conditional function for determining when the wait_for_msgs_and_events loop is complete.
+ *
+ * @param msg_persist_ready true if a msg has ever been received
+ * @param subscription_persist_readytrue if a subscription has been received
+ * @param publisher_persist_ready true if a pulbisher event has been received
+ * @return true when the desired conditions are met
+ */
+using ConditionalFunction = std::function<
```

Comment:
Naming is really inconsequential here, but I'd call this a predicate.

Reply:
Fixed.